test/runner.py

```diff
@@ -566,6 +567,16 @@ def set_env(name, option_value):
   utils.delete_file(common.flaky_tests_log_filename)
   utils.delete_file(common.browser_spawn_lock_filename)
   utils.delete_file(f'{common.browser_spawn_lock_filename}_counter')
+  if options.force_browser_process_termination:
+    config = common.get_browser_config()
+    def terminate_all_browser_processes():
+      procs = common.list_processes_by_name(config.executable_name)
+      if len(procs) > 0:
+        print(f'Terminating {len(procs)} stray browser processes.')
+        common.terminate_list_of_processes(procs)
+
+    atexit.register(terminate_all_browser_processes)
+    terminate_all_browser_processes()
```

Comment:
I wish this kind of code could live in the setUpClass of the BrowserCore, but I guess that problem with that is we don't want to run this on each parallel worker?

Reply:
Its kind of unfortunate (and maybe a bug) that `setUpClass` is called more then once per per run.. maybe we should add some kind of `setUpClassOncePerRun` helper :)